Firmware/MotorControl/controller.hpp

```diff
@@ -75,17 +76,17 @@ class Controller : public ODriveIntf::ControllerIntf {
     bool* pos_estimate_valid_src_ = nullptr;
     float* vel_estimate_src_ = nullptr;
     bool* vel_estimate_valid_src_ = nullptr;
-    int32_t* pos_wrap_src_ = nullptr; // enables circular position setpoints if not null. The value pointed to is the maximum position value.
+    float* pos_wrap_src_ = nullptr; // enables circular position setpoints if not null. The value pointed to is the maximum position value.
```

Comment:
"enables circular position setpoints if not null" This logic was changed, you can probably just delete that part.

Reply:
Fixed in a262688